doc/source/license.rst

```diff
@@ -1,40 +1,39 @@
-License
-=======
+:tocdepth: -1
+
+.. index:: license
 
-This program is part of the DiffPy open-source project at Columbia
-University and is available subject to the conditions and terms laid out 
-below.
+License
+#######
 
-Copyright © 2009-2019, Trustees of Columbia University in the City of 
-New York, all rights reserved.
+OPEN SOURCE LICENSE AGREEMENT
+=============================
+BSD 3-Clause License
 
-For more information please visit the diffpy web-page at 
-http://diffpy.org or email Prof. Simon Billinge at sb2896@columbia.edu.
+Copyright (c) 2009-2024, The Trustees of Columbia University in 
```

Comment:
Added 2009

Reply:
That seems a little early for snmf.  Are you sure about that?